spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -279,6 +282,45 @@
     end
   end
 
+  let(:creative_commons_attribution_v3_url) { "http://creativecommons.org/licenses/by/3.0/us/" }
+  let(:expected_creative_commons_attribution_v3_url) do
+    {
+      "rights": "Creative Commons Attribution 3.0 United States",
+      "scheme": "spdx",
+      "identifier": "CC-BY-3.0-US",
+      "url": creative_commons_attribution_v3_url
+    }
+  end
+
+  let(:creative_commons_zero_url) { "http://creativecommons.org/publicdomain/zero/1.0/" }
+  let(:expected_mit) do
+    {
+      "rights": "Creative Commons Zero v1.0 Universal",
+      "scheme": "spdx",
+      "identifier": "CC0-1.0",
+      "url": creative_commons_zero_url
+    }
+  end
+
+
+  let(:mit_license_url) { "https://opensource.org/licenses/MIT" }
+  let(:expected_mit) do
+    {
+      "rights": "MIT License",
+      "scheme": "spdx",
+      "identifier": "MIT",
+      "url": mit_license_url
+    }
+  end
+
+  describe "#rights" do
```

Comment:
It's not the most helpful spec, but I didn't think it hurt to add a few more examples.